engine/src/main/java/org/hibernate/search/engine/search/predicate/dsl/SearchPredicateFactory.java

```diff
@@ -82,6 +83,55 @@ public interface SearchPredicateFactory {
 	@Deprecated
 	PredicateFinalStep bool(Consumer<? super BooleanPredicateClausesStep<?>> clauseContributor);
 
+	/**
+	 * Match documents if they match a conjunction of boolean clauses.
```

Comment:
While you are perfectly right that "conjunction"/"disjunction" are the correct words, I suspect some people will not understand it immediately. For example, think of non-native English speakers whose native language is fundamentally different, e.g. Mandarin. Or more likely, students, even native English speakers, who haven't been studying hard enough :)

In any case, if there's a way to make this documentation simpler, let's do it?

```suggestion
	 * Match documents if they match all inner clauses.
```

Same for the other methods in this class, of course.

Reply:
Yup, I was also wondering about that...
So conjunction -> all, disjunction -> any!